src/db/schema.ts

```
@@ -155,13 +155,36 @@ export const insertResourceSchema = createInsertSchema(resources, {
   resourceType: z.enum(["video", "article"]),
   categoryName: z.string().min(1),
   url: z.string().url(),
+  // image: z
+  //   .custom<File>((file) => file instanceof File, "Invalid image file")
+  //   .optional(),
+  image: z
+    .custom<File | undefined>((file) => {
+      if (!file) return true; // Allow no image (optional)
```

ESLint

[error] Expect newline after if

(antfu/if-newline)